### PR TITLE
Added initial docker image

### PR DESCRIPTION
**Pull request by sergeyklay**
/cc @ruudboon, @Jeckerson

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# debian/buster
+FROM php:7.3-cli
+
```
Comment (ruudboon) [inline Dockerfile]: Can we add some labels?
LABEL vendor="Phalcon"
LABEL maintainer="team@phalcon.io"

**Comment by ruudboon** [comment]
Can we move the Dockerfile to the resource dir? https://github.com/php-pds/skeleton/tree/1.0.0#summary

**Comment by sergeyklay** [comment]
@ruudboon done

**Comment by sergeyklay** [comment]
@ruudboon
> Still missing the labels. 

Which ones?
https://github.com/phalcon/vokuro/blob/e4050ecfd68f2d5891f83418452019bbc0c3c8f6/resources/Dockerfile#L15-L20

**Comment by ruudboon** [comment]
@sergeyklay Sorry missed it! 🤐

**Pull request merged**